ChangeLog

```diff
@@ -2,10 +2,6 @@ CHANGES
 =======
 
 * Convert README to RST
-
-0.2.3
------
-
 * Implement PBR for versioning
```

Comment:
this change is being made by PBR because the latest git tags aren't fetched from master.  You can pull them in and update the commit, or just take this change out.